kubescape image scan panics with index out of range on registry images without an organization path

# Description

`getAttributesFromImage` in `core/core/image_scan.go` normalizes the image name via `cautils.NormalizeImageName` and then does:

```go
tokens := strings.Split(canonicalImageName, "/")
registry := tokens[0]
organization := tokens[1]
imageNameAndTag := strings.Split(tokens[2], ":")
```

This assumes the canonical name always has exactly 3 path components (`registry/org/image:tag`). That assumption is false. The `distribution/reference` library only injects the `library/` namespace for bare Docker Hub names. Any image from a custom or self-hosted registry without an explicit namespace normalizes to 2 components, so `tokens[2]` is out of range and the program panics.

Additionally, `getUniqueVulnerabilitiesAndSeverities` ignores the error returned from `getAttributesFromImage`. It logs the error via `StopError`, which does not exit the process, and then continues with zero-value attributes. Wildcard or permissive exception targets then match the empty attribute set, silently suppressing real CVEs from the scan.

# Environment

OS: `macOS Darwin (arm64)`
Version: `master (820fa314)`

# Steps To Reproduce

1. Run:

```bash
kubescape image scan registry.example.com/myapp:1.0 --exceptions exc.json
```

where `exc.json` is any valid exceptions file (or even `{}`).

2. The CLI panics before any registry or network access happens.

Verified normalization output:
registry.example.com/myapp:1.0  -> registry.example.com/myapp:1.0  (2 tokens)
localhost:5000/myimage          -> localhost:5000/myimage          (2 tokens)
quay.io/kubescape/cli:v3        -> quay.io/kubescape/cli:v3        (3 tokens)
ghcr.io/org/team/app:v1         -> ghcr.io/org/team/app:v1         (4 tokens)

# Expected behavior

`kubescape image scan` should parse any valid image reference (with or without a registry namespace, with ports, with multi-segment paths, or with digests) without crashing, and should not apply exception filters when the image reference cannot be parsed.

# Actual Behavior

panic: runtime error: index out of range [2] with length 2
The CLI crashes with an unrecoverable panic for any image referenced without an explicit namespace segment. The `--exceptions` feature is effectively unusable for such images.

Additionally, images with more than 3 path components (e.g. `ghcr.io/org/team/app:v1`) don't crash but are silently mis-parsed: `ImageName` becomes `"team"` and `ImageTag` becomes `"latest"` instead of `"app"` / `"v1"`, so exception matching silently targets the wrong image.

I have a fix ready and can open a PR with tests.

--- a/core/core/image_scan.go
+++ b/core/core/image_scan.go
@@ -67,11 +67,20 @@ func getAttributesFromImage(imgName string) (Attributes, error) {
 		return Attributes{}, err
 	}
 
+	// canonicalImageName is registry/[path/...]/name[:tag]. The registry is always
+	// the first component, but the organization path in between is optional (a
+	// registry image can have no organization) and may span multiple segments, so
+	// don't assume a fixed three-token split. See #2391.
 	tokens := strings.Split(canonicalImageName, "/")
 	registry := tokens[0]
-	organization := tokens[1]
 
-	imageNameAndTag := strings.Split(tokens[2], ":")
+	organization := ""
+	nameAndTag := tokens[len(tokens)-1]
+	if len(tokens) > 2 {
+		organization = strings.Join(tokens[1:len(tokens)-1], "/")
+	}
+
+	imageNameAndTag := strings.Split(nameAndTag, ":")
 	imageName := imageNameAndTag[0]
 
 	// Intialize the image tag with default value

--- a/core/core/image_scan_test.go
+++ b/core/core/image_scan_test.go
@@ -127,6 +127,39 @@ func TestGetAttributesFromImage(t *testing.T) {
 			},
 			expectedErr: nil,
 		},
+		{
+			// registry image with no organization path (previously panicked)
+			imageName: "myregistry.io/myimage:v1",
+			expectedAttributes: Attributes{
+				Registry:     "myregistry.io",
+				Organization: "",
+				ImageName:    "myimage",
+				ImageTag:     "v1",
+			},
+			expectedErr: nil,
+		},
+		{
+			// registry with a port, no organization, default tag
+			imageName: "localhost:5000/myimage",
+			expectedAttributes: Attributes{
+				Registry:     "localhost:5000",
+				Organization: "",
+				ImageName:    "myimage",
+				ImageTag:     "latest",
+			},
+			expectedErr: nil,
+		},
+		{
+			// multi-segment organization path
+			imageName: "gcr.io/team/sub/myimage:v2",
+			expectedAttributes: Attributes{
+				Registry:     "gcr.io",
+				Organization: "team/sub",
+				ImageName:    "myimage",
+				ImageTag:     "v2",
+			},
+			expectedErr: nil,
+		},
 	}
 
 	for _, tt := range tests {

